### PR TITLE
Turn off NSF sponsorship

### PR DESCRIPTION
**Pull request by daisieh**
Removes the wording about possible NSF sponsorship as well as the intervention screen during checkout.

The funder concept still needs to be updated to not have a subscription plan or customerID on the production system.

### CODE REVIEW

**Comment by daisieh** [comment]
To test:
(while NSF funder concept is still sponsored)
Make submissions with journal with subscription plan and:
- no grant
- invalid grant
- valid grant

In all cases, journal subscription should take effect.
Make submissions with journal w/o subscription plan and the same grant options. For the valid grant, NSF should show as the sponsor. No one should sponsor in the other cases.

Remove the NSF funder concept subscriptionPlan and customerID metadata. Then try a submission with non-sponsored journal and valid NSF grant. No one should sponsor.

**Comment by ryscher** [comment]
Initial submission page still says "Funding Agency Sponsorship"

File: dspace/modules/api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java [hunk 1]
```diff
@@ -309,27 +311,49 @@ public boolean isSponsored(Context context, ShoppingCart shoppingcart) throws SQ
...
                 if (journal != null && journal.length() > 0) {
-                    //update shoppingcart
                     DryadJournalConcept journalConcept = JournalUtils.getJournalConceptByJournalName(journal);
                     if (journalConcept != null) {
                         shoppingcart.setSponsoringOrganization(journalConcept);
                     }
                 }
+
+                // funder of last resort:
+                log.error("checking to see if " + funder + " is a sponsor");
```
Comment (ryscher) [inline dspace/modules/api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java]: Not an error -- this should be log.info

File: dspace/modules/api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java [hunk 2]
```diff
@@ -309,27 +311,49 @@ public boolean isSponsored(Context context, ShoppingCart shoppingcart) throws SQ
...
+                if (!shoppingcart.hasSubscription()) {
+                    if (!"".equals(funder)) {
+                        DryadFunderConcept funderConcept = DryadFunderConcept.getFunderConceptMatchingFunderID(context, funder);
+                        if (funderConcept != null && funderConcept.getSubscriptionPaid()) {
+                            log.error("funder is a sponsor");
```
Comment (ryscher) [inline dspace/modules/api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java]: Not an error -- this should be log.info

**Review by ryscher — APPROVED** [review]
Initial submission page still says "Funding Agency Sponsorship"

**Pull request merged**